packages/integration/lib/src/views/compose/compose_screen.dart

```diff
@@ -434,18 +437,44 @@ class _LMFeedComposeScreenState extends State<LMFeedComposeScreen> {
                 Widget mediaWidget;
                 switch (composeBloc.postMedia[index].mediaType) {
                   case LMMediaType.image:
-                    mediaWidget = LMFeedImage(
-                      imageFile: composeBloc.postMedia[index].mediaFile,
-                      style: style?.mediaStyle?.imageStyle,
-                      position: index,
+                    mediaWidget = Container(
```

Comment:
if we pass container widget than how we can access copyWith() method on this?

Reply:
We aren't using copywith in these segments, the type of mediaWidget is Widget, so we cannot use copyWith anyway